src/workerd/api/http.h

```diff
@@ -471,9 +471,15 @@ class Fetcher: public jsg::Object {
   jsg::Promise<QueueResponse> queue(
       jsg::Lock& js, kj::String queueName, kj::Array<ServiceBindingQueueMessage> messages);
 
+  jsg::Promise<kj::Maybe<jsg::V8Ref<v8::Value>>> getRpcAndCallRemote(
+      jsg::Lock& js, kj::String methodName, jsg::Optional<kj::Array<jsg::Value>> args=nullptr);
```

Comment:
Nit: Having the last argument be a `jsg::Varargs` would likely be better?

Reply:
Didn't address this yet; wanted to get some changes up before going deeper into capnp stuff. Will revisit!

Reply:
Fwiw, since leaving the comment I rediscovered the `jsg::Arguments<T>` mechanism that kenton added. That would likely be better here.